torch/backends/cusparselt/__init__.py

```diff
@@ -25,12 +25,12 @@ def _init():
         global __MAX_ALG_ID
         if __cusparselt_version is None:
             __cusparselt_version = _cusparselt.getVersionInt()
-            if __cusparselt_version == 400:
-                __MAX_ALG_ID = 4
-            elif __cusparselt_version == 502:
-                __MAX_ALG_ID = 5
-            elif __cusparselt_version == 602:
-                __MAX_ALG_ID = 37
+
+            # only way to get MAX_ALG_ID is to run a matmul
+            A = torch.zeros(128, 128, dtype=torch.float16).cuda()
```

Comment:
Is this single __MAX_ALG_ID used for all data types, bias, layouts, activation? If yes, this is not good since it depends on each of these features and therefore MatmulPlan and is not a constant.